nginx.conf.template

```diff
@@ -117,17 +97,18 @@ http {
     location / {
       proxy_ignore_client_abort on;
       proxy_set_header X-Real-IP  $remote_addr;
-      proxy_set_header Host FRONTEND_DNS_NAME;
       proxy_set_header X-3scale-proxy-secret-token $secret_token;
 
       # limit requests from the same client, allow `burst` to 20 r/s,
       # `nodelay` or drop connection immediately in case it exceeds this
       # threshold.
       limit_req zone=one burst=20 nodelay;
 
-      if ($request_method = GET ) {
-        proxy_pass http://$bdb_backend:BIGCHAINDB_BACKEND_PORT;
-      }
+      # We do not need the GET handling here as it's done in the other NGINX
+      # module
```

Comment:
So this NGINX (OPENRESTY) only handles the POST requests? (That would be okay, I'm just asking to verify.)

If so, maybe do a search for the string "GET" in this file to make sure you didn't miss something.